dev-libs/libprelude/libprelude-4.0.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+USE_RUBY="ruby21 ruby22 ruby23 ruby24"
+
+DISTUTILS_OPTIONAL=1
+
+inherit autotools distutils-r1 ruby-single
+
+DESCRIPTION="Prelude-SIEM Framework Library"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="lua perl python ruby"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+RDEPEND="net-libs/gnutls:=
+	lua? ( dev-lang/lua:* )
+	perl? ( dev-lang/perl:= virtual/perl-ExtUtils-MakeMaker )
+	python? ( ${PYTHON_DEPS} )
+	ruby? ( ${RUBY_DEPS} )"
+
+DEPEND="${RDEPEND}
+	sys-devel/flex
+	virtual/yacc
+	>=dev-lang/swig-3.0.7
+	virtual/pkgconfig
+	dev-util/gtk-doc-am"
+
+PATCHES=(
+	"${FILESDIR}/${P}-fix-python-bindings.patch"
+)
+
+python_compile() {
+    cd bindings/python || die
+	distutils-r1_python_compile
+}
+
+python_install() {
+    cd bindings/python || die
+	distutils-r1_python_install
+}
+
+src_prepare() {
+	default
+
+	# Avoid null runpaths in Perl bindings.
+	sed -e 's/ LD_RUN_PATH=""//' -i "${S}/bindings/Makefile.am" || die "sed failed"
+
+	eautoreconf
+
+	if use python; then
+		distutils-r1_src_prepare
+	fi
+}
+
+src_configure() {
+	econf \
+		--enable-easy-bindings \
+		--with-swig \
+		$(use_with lua) \
+		$(use_with perl) \
+		$(usex perl '--with-perl-installdirs=vendor' '' ) \
+		$(use_with python python2) \
+		$(use_with python python3) \
+		$(use_with ruby)
+
+	if use python; then
+		distutils-r1_src_configure
```

Comment:
This does nothing. See the [colorful table](https://wiki.gentoo.org/wiki/Project:Python/distutils-r1#Phase_functions).

Reply:
ok